core/src/scss/utilities/mixins/link/_link.scss

```diff
@@ -9,13 +9,18 @@
 /// @param {variable} $hover - Default hover color
 ///
 /// @group mixin
-@mixin link($color: color(link), $hover: color(link--hover)) {
+@mixin link(
+  $color: color(link),
+  $hover: color(link--hover),
+  $focus: color(focus)
+) {
+  @include focus-visible($focus);
+  @include padding(null 0.15rem);
```

Comment:
It looks like the generic `<a>` element uses this `@link` mixin, this adds left/right adding to all the links (which could be everywhere - nav, lockup etc. This could be a little dangerous. I understand how it might look better with more spacing around if we're adding focus shadow, but I'm a little bit worried about this change as it affects all the links.

Reply:
Good point, @yvonnetangsu !

This was something @ishahism had requested. It's very subtle. 

I wonder if @kerri-augenstein or @katrialesser would like to weigh in here?